frameworks/hdfs/tests/config.py

```diff
@@ -24,7 +24,7 @@
 KEYTAB = "hdfs.keytab"
 HADOOP_VERSION = "hadoop-2.6.0-cdh5.9.1"
 
-DOCKER_IMAGE_NAME = "elezar/hdfs-client:dev"
+DOCKER_IMAGE_NAME = "mesosphere/hdfs-testing-client:30ff636fdf1795a7c3f443cfa77898720a3e6fd3"
```

Comment:
Nice, I can also update the client we're using [here](https://github.com/mesosphere/soak-cluster-configurations/blob/master/infinity/packages/hdfsclient-kerberos.json#L8) now

Reply:
@samvantran the client here: `mesosphere/hdfs-testing-client:6972ea3833c9449111aceaa998e3e093a9c8dcee` works as expected!